reporters_db/data/reporters.json

```diff
@@ -3584,7 +3590,9 @@
                 "us:nc;supreme.court"
             ],
             "name": "Carolina Law Repository (N. C.)",
-            "variations": {}
+            "variations": {
+                "Car. Law. Repos.": "Car. Law Repos."
```

Comment:
this is a typo ... not a variation no?

Reply:
Not 100% sure, i believe the citation has a dot in the reporter, the json has this citation: 1 Car. Law. Repos. 346

![image](https://user-images.githubusercontent.com/6474994/188011181-366926dc-cf73-4fe4-b47b-51c2a90af53a.png)

I believe it's a dot, what do you think?

Reply:
proves me wrong.